snuba_sdk/column.py

```diff
@@ -72,3 +72,8 @@ def validate_data_model(self, entity: Entity) -> None:
             raise InvalidColumnError(
                 f"entity '{entity.name}' does not support the column '{self.name}'"
             )
+
+
+@dataclass(frozen=True)
+class Variable(Column):
```

Comment:
Why did you add this class?

Reply:
Was playing around with adding support for variables in expressions (e.g. `$foo`). Will remove for this PR.